### ISSUE TITLE
Playlist metadata with 100+ videos

### ISSUE THREAD

**Issue by talmobi** [talmobi/yt-search#45]
Currently getting playlist metadata of a playlist id is capped at 100 videos even if the playlist has more than 100 videos in it. This is because `ytInitialData` only includes the first 100 videos in the playlist.

This issue is to track/discuss if or how this library should handle this case.

related issue: https://github.com/talmobi/yt-search/issues/44

**Comment by xakiru** [comment]
I think it's 200 now, or atleast that's what I get from checking this var ytInitialData.contents.twoColumnWatchNextResults.playlist.playlist

**Comment by undaBen** [comment]
I am using the version 2.8.0 and it still capping at 100

**Comment by KriskotooBG** [comment]
I believe that this would be a very good addition, as most playlists I'm working with have more than 100 items in them. Maybe some kind of paging/limiting system might be good , for example you can set a limit of 200 items, or more, or count in pages, example page one, page two etc.

**Comment by talmobi** [comment]
@KriskotooBG I agree it would be a powerful feature. But I would like to point out that this package isn't intended to be a high-throughput crawler -- you will be bound by youtube's internal throttling at the very least.

The main focus of this package is for simple "get the info you would see as a user visiting/searching YouTube in a browser".

It's also a bit complex to do as there are a few different ways of doing it. And I don't have the time to spend thinking about this right now or developing it. So I am hesitant of adding this.

However on the README there is a "Similar" section that mentions https://www.npmjs.com/package/ytsr that should, if I recall correctly, support this feature.
